recipes/gstreamer/all/conanfile.py

```diff
@@ -70,6 +70,7 @@ def _configure_meson(self):
         meson.options["examples"] = "disabled"
         meson.options["benchmarks"] = "disabled"
         meson.options["tests"] = "disabled"
+        meson.options["introspection"] = "disabled"
```

Comment:
Did you create issues for those "multiple issues"? If not, please do

Reply:
@Croydon I haven't done so yet. I'll try to bring together all packages (e.g. gstreamer, gdk-pixbuf) which fail introspection generation to one issue as the errors seem similar on first look.

Reply:
Then please mark the pull request as drafts, it would be weird if we merge something and get to know the errors only afterwards.

Reply:
See issue #6726